test/e2e/components/vl-map-search.js

```diff
@@ -0,0 +1,50 @@
+const { VlElement } = require('vl-ui-core').Test;
+const { VlSelect } = require('vl-ui-select').Test;
+const { By } = require('vl-ui-core').Test.Setup;
+
+class VlMapSearch extends VlElement {
+
+    async _getSearch() {
+        return this.shadowRoot;
+    }
+
+    async _getSelect() {
+        const search = await this._getSearch();
+        const select = await search.findElement(By.css('select[is="vl-select"]'));
+        return new VlSelect(this.driver, select);
+    }
+
+    async open() {
+        const select = await this._getSelect();
+        await select.open();
+    }
+
+    async search(text) {
+        const search = await this._getSearch();
+        const input = await search.findElement(By.css('.vl-select__list > input'));
+        await this.driver.executeScript(`arguments[0].value='${text}'`, input);
+        const select = await this._getSelect();
+        await this.driver.executeScript(`arguments[0].dispatchEvent(new CustomEvent('search', {detail: {value: '${text}'}}))`, select);
```

Comment:
Via gewone browser input lukte het niet? Want het is nu niet echt hoe een eindgebruiker een zoekopdracht zou uitvoeren.

Reply:
Nee, sendKeys gaf een probleem (element not interactable of zo), en dacht toch dat ik het juiste object had. Zag niet direct iets beter dan dit helaas.

Reply:
In Firefox? Want da's hetzelfde probleem als dat we bij de datepicker hadden, maar enkel in Firefox.

Reply:
Dit was in chrome

Reply:
https://www.milieuinfo.be/jira/browse/UIG-603